x-pack/plugins/reporting/server/lib/event_logger/logger.ts

```diff
@@ -27,9 +28,39 @@ export interface ExecutionCompleteMetrics {
   byteSize: number;
 }
 
+const getEventLog = (logger: LevelLogger) => ({
+  getLogger(lProperties: Partial<IEvent>): IEventLogger {
+    return (function () {
+      let start: Date | undefined;
+      let end: Date | undefined;
+      let duration: number | undefined;
+
+      return {
+        logEvent(properties: IEvent) {
+          if (start && !end) {
+            end = start ? new Date() : undefined;
+          }
+          duration = end && start ? end.valueOf() - start.valueOf() : undefined;
+          const timing: Partial<IEvent> = {
+            event: { duration, start: start?.toISOString(), end: end?.toISOString() },
+          };
+          const mProperties: Partial<IEvent> = deepMerge(lProperties, timing);
+          logger.debug(JSON.stringify(deepMerge(mProperties, properties)), ['reporting-event-log']);
```

Comment:
This tag controls how users can filter for the event data. Maybe the tag should be something more future-proof than `reporting-event-log`

Reply:
I don't think I'd put `log` in there, but `reporting-event` seems appropriate ...